upi/openstack/05_compute-nodes.yaml

```diff
@@ -0,0 +1,40 @@
+# Required Python packages:
+#
+# ansible
+# openstacksdk
+# netaddr
+
+- hosts: all
+
+  vars:
+    compute_data: "worker.ign"
+
+    compute_server: "{{ os_infra_id }}-worker"
+    compute_port: "{{ os_infra_id }}-worker-port"
+
+    # These computed lists are used to feed the Ansible loops with the number
+    # of iterations specified in the Inventory.
+    compute_servers: "{{ [compute_server] * os_compute_nodes_number }}"
+    compute_ports: "{{ [compute_port] * os_compute_nodes_number }}"
+
+  tasks:
+  - name: 'Create the Compute ports'
+    os_port:
+      name: "{{ item.1 }}-{{ item.0 + 1 }}"
```

Comment:
Just like in the control plane, all the indices should be zero-based for consistency with IPI and current UPI doc, I think.